### PR TITLE
35 add network cascading interdependency analysis

### PR DESCRIPTION
**Pull request by nunezco2**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## [Unrelease] 
+## [3.6.0] - 2022-09-14
```
Comment (longshuicy) [inline CHANGELOG.md:8]: didn't we save that for the release branch?

**Review by longshuicy — APPROVED** [review]

File: datatypes/analyses_hazard_other.md [hunk 2]
```diff
@@ -56,16 +56,18 @@ datasets because they are used in active analyses
 - incore:Employment
 - incore:epf
 - incore:epfDamage
+- incore:epfDiscretizedCascadingFunc
 - incore:epnLinkeVer1
 - incore:epnNodeVer1
+- incore:epnNetwork
```
Comment (ywkim312) [inline datatypes/analyses_hazard_other.md:62]: is epnNetwork belonged in here?

Comment (nunezco2) [reply datatypes/analyses_hazard_other.md:62]: It should belong here

Comment (ywkim312) [reply datatypes/analyses_hazard_other.md:62]: got it!

**Review by ywkim312 — APPROVED** [review]

**Pull request merged**